core-integration-test/src/test/groovy/org/openstack4j/api/AbstractSpec.groovy

```diff
@@ -18,7 +18,7 @@ import co.freeside.betamax.tape.yaml.TapePropertyUtils
 abstract class AbstractSpec extends Specification {
 
     def static String MODULEROOT = Paths.get("").toAbsolutePath().getParent().toString()
-    def static String TAPEROOT = "/" + MODULEROOT + "src/test/resources/tapes/"
+    def static String TAPEROOT = MODULEROOT + "/" + "src/test/resources/tapes/"
```

Comment:
Yes, makes sense.

The remaining 3 issues, which prevent os4j from going back to green, are still unclear to me. 
Maybe it's something in Betamax. Need to dig in this again tomorrow - unless you're faster.